optuna_dashboard/ts/components/TrialTable.tsx

```diff
@@ -49,8 +56,9 @@ export const TrialTable: FC<{ studyDetail: StudyDetail }> = ({
       const result = await trialFilter(studyDetail.trials, filterQuery)
       setFilteredTrials(result)
     } catch (error) {
-      console.error("Filter failed:", error)
-      handleClearFilter()
+      // eslint-disable-next-line no-empty
+      // Error handling is delegated to onDenied/onFailed callbacks to avoid
+      // emmiting error logs when user denied the execution.
```

Comment:
[Note] I've confirmed that the error logging was suppressed when users pushed the deny button. This behavior is natural and expected when users intentionally perform a rejection action.